frontend/src/features/permits/types/ApplicationRejectionHistory.ts

```diff
@@ -0,0 +1,8 @@
+import { Dayjs } from "dayjs";
+
+export interface ApplicationRejectionHistory {
+  caseActivityId: number;
+  userName: string;
+  dateTime: Dayjs;
```

Comment:
We need to be careful of a nuance for such datetime types inside Application/Permit types. When we receive the data from the endpoint (eg. ApplicationResponseData, as a dto), that's when this value is actually a string. We would need to deserialize it (with explicit logic to convert the string into a Dayjs object) in order for this value to be of type Dayjs

Reply:
Similarly, when we create/update an application/permit, we would need to make sure to serialize the application data (ie. with explicit logic to convert Dayjs to string) before sending it off the the backend as the request payload, since Dayjs is not a primitive type and therefore cannot be automatically serialized as part of json

Reply:
See the existing logic for startDate and expiryDate for more on how datetime types/values are used, and how and when they are used and need to be serialized/deserialized, and do the same for this field